Makefile

```diff
@@ -25,13 +25,11 @@ rebuild-lockfiles: .pdm
 
 .PHONY: format  ## Auto-format python source files
 format: .pdm
-	pdm run black $(sources)
 	pdm run ruff --fix $(sources)
```

Comment:
```suggestion
	pdm run ruff format $(sources)
```

Reply:
Change for line 32 as well?

Reply:
@Luca-Blight I think we want to leave line 32 as is, as you call the [ruff linter](https://docs.astral.sh/ruff/linter/) with slightly different syntax compared to the [ruff formatter](https://docs.astral.sh/ruff/formatter/)